rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -431,16 +509,63 @@ def _build_tf_train_graph(self) -> Tuple["tf.Tensor", "tf.Tensor"]:
             self.scale_loss,
         )
 
-    def _build_tf_pred_graph(
-        self, session_data: "train_utils.SessionData"
-    ) -> "tf.Tensor":
+    def combine_sparse_dense_features(
+        self,
+        batch: Dict[Text, tf.Tensor],
+        key_prefix: Text,
```

Comment:
I think it is better to directly pass a list of tensors, then we apply sparse_dense layer if it is `SparseTensor` and nothing if it is normal tensor, then we use `reduce_mean` on all resulting dense layers. If we use mean for word vectors, I think it makes sense to use mean on sparse_dense layer output. The difference is scaling factor, but it is way less confusing if we use the same thing everywhere